lib/util/cache/run.ts

```diff
@@ -1,13 +1,15 @@
-let repoCache: Record<string, any> = {};
+let repoCache: Record<string, any>;
 
-export function clear(): void {
+export function init(): void {
   repoCache = {};
 }
```

Comment:
```suggestion
export function init(): void {
  repoCache = {};
}
export function reset(): void {
  repoCache = undefined;
}
```

Function to unset cache for tests